src/js/utils/merge-options.js

```diff
@@ -41,7 +43,7 @@ const customizer = function(destination, source) {
  * provided objects
  * @function mergeOptions
  */
-export default function mergeOptions() {
+mergeOptions = function() {
```

Comment:
should probably make this `const mergeOptions` here and get rid of the let above